exporter/datadogexporter/translate_traces.go

```diff
@@ -272,33 +242,43 @@ func spanToDatadogSpan(s pdata.Span,
 	}
 
 	// Set Span Status and any response or error details
-	status := s.Status()
-	if !status.IsNil() {
-		// map to known msg + status
-		code, ok := statusCodes[int32(status.Code())]
-		if !ok {
-			code = codeDetails{
-				message: "ERR_CODE_" + strconv.FormatInt(int64(status.Code()), 10),
-				status:  http.StatusInternalServerError,
-			}
+	if status := s.Status(); !status.IsNil() {
+		isError := 0
```

Comment:
Why not use a boolean value for `isError`?

Reply:
datadog `Span.Error` has to be a 32bit int of either 1/0, so i tried to combine some things here. I've updated this to use constants so that it's a bit easier to read and the if statements are cleaner